contracts/JBTiered721DelegateDeployer.sol

```diff
@@ -17,8 +19,6 @@ import './JB721GlobalGovernance.sol';
 contract JBTiered721DelegateDeployer is IJBTiered721DelegateDeployer {
   error INVALID_GOVERNANCE_TYPE();
 
-  uint256 constant DEPLOY_BYTECODE_LENGTH = 13;
```

Comment:
curious where this "13" is in the OZ extension https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/Clones.sol

Reply:
Ooh, I think it's a leftover of our cloning pattern, will remove it, good catch

Reply:
![image](https://user-images.githubusercontent.com/83670532/208923722-b718359f-6f62-4c8f-b64f-a136f76a13b5.png)

So basically, 13 is the equivalent of the 0x0a (red arrows) which say "hey, the bytecode to keep onchain is at the offset 0a", meaning the constructor is the green box only

Reply:
got it. cool.

Reply:
(to be more precise, first we copy the code which starts at 0x0a and is 0x2d long in memory, then we return the content of the memory address 0x0 to 0x2d)